.scalafmt.conf

```
@@ -1 +1 @@
-version = "3.0.8"
+version = "3.3.1"
```

```suggestion
version = "3.3.1"
runner.dialect = scala213
```

https://github.com/scalapb/protobuf-scala-runtime/runs/4683368618?check_suite_focus=true#step:5:127

```
[error] org.scalafmt.sbt.ScalafmtSbtReporter$ScalafmtSbtError: scalafmt: Invalid config: Default dialect is deprecated; use explicit: [sbt0137,sbt1,scala211,scala212,scala212source3,scala213,scala213source3,scala3]
[error] Also see https://scalameta.org/scalafmt/docs/configuration.html#scala-dialects"
[error]  [/home/runner/work/protobuf-scala-runtime/protobuf-scala-runtime/.scalafmt.conf]
```